.reek

```diff
@@ -30,6 +30,8 @@ UncommunicativeMethodName:
   exclude:
     - PhoneConfirmationFlow
     - render_401
+    - SessionDecorator#registration_bullet_1
+    - ServiceProviderSessionDecorator#registration_bullet_1
```

Comment:
I am not getting an UncommunicativeMethodName warning for these. I wonder it's a Code Climate vs us issue?